PRChecker/Shared/Extensions/Array+Extensions.swift

```diff
@@ -8,7 +8,7 @@
 import Foundation
 
 extension Array where Element: Equatable {
-    func removingLaterDuplicates() -> [Element] {
+    func arrayByRemovingDuplicates() -> [Element] {
```

Comment:
Maybe we could consider using an [`OrderedSet`](https://github.com/apple/swift-collections/blob/main/Documentation/OrderedSet.md) lol

Reply:
Yes, that's exactly what I want, but I'm not sure if its worth importing the whole package just for that